pkg/key/key.go

```diff
@@ -16,7 +16,7 @@ const (
 	// Release is the latest release version we use as base for conformance
 	// testing. This version changes with each new release so that we stay
 	// synchronized with the latest AWS release we publish.
-	Release = "12.0.0"
+	Release = "11.5.0"
```

Comment:
I was a bit too eager with the latest release version. During cluster creation testing I noticed the latest release is something else still.